content/guides/roles-and-permissions.md

```diff
@@ -3,3 +3,29 @@ title: "Roles and permissions"
 metaTitle: "sensenet admin-ui - Roles and permissions"
 metaDescription: "sensenet admin-ui - Roles and permissions"
 ---
+
+User management is a core function of the business, that includes user creation, rights management and assigning roles to the users. Effective user management contains adjusting the access rights of the users who change role, get promoted or leave the company. Usually it's important to make these changes in short time and this way we can increase both effectiveness and integrity.
+
+# Built-in and custom permissions
+There is a set of basic built-in permissions, such as See, Open, Save, Delete etc., and it is even possible to add custom ones as well.
+
+**See**: you only have the information that the content exists (content metadata cannot be accessed)
+**Open**: you have access to content metadata
+**Save**: you can make changes in the content metadata
+
+Other permissions for removing or approving a content or managing lists and workspaces.
+as mentioned above, the built-in permission set can be extended with custom ones as well.
+
+# Built-in roles for SNaaS users
+**Administrators**: Admins are considered to be the most powerful users in the system. This may be true for some features but not necessarily for all content. In the default structure, administrators have access to everything, but when you build your project and add your custom content, it is perfectly fine if you hide something from the Administrators group. For example you may remove their permissions from confidential documents completely.
```

Comment:
```suggestion
**Administrators**: are considered to be the most powerful users in the system. This may be true for some features but not necessarily for all content. In the default structure, administrators have access to everything, but when you build your project and add your custom content, it is perfectly fine if you hide something from the Administrators group. For example you may remove their permissions from confidential documents completely.
```